lib/lita/handlers/cmd.rb

```diff
@@ -37,6 +38,9 @@ def run_action(resp)
 
         script_path = "#{config.scripts_dir}/#{script}"
         env_vars    = { 'LITA_USER' => resp.user.name }
+        script_group = script.split('/')[0]
+        cred = resp.user.metadata["cred-#{script_group}"]
```

Comment:
I assume that you set the user metadata for the group here out of band. Is this what you meant by adding in a Redis key? I'm unfamiliar with the metadata loading system. What is the Redis key that you modify?